src/transaction/submit.ts

```diff
@@ -1,5 +1,6 @@
 // Deprecated - use client.request instead:
-//   const response = await client.request('submit', {
+//   const response = await client.request({
+//     command: 'submit',
 //     tx_blob: signedTransaction,
 //     fail_hard: failHard
 //   });
```

Comment:
Should this code be removed?

Reply:
It's a little out of the scope of this PR, and this whole folder is going to be basically cleared out anyways.